spec/acceptance/nodesets/fedora21.yml

```diff
@@ -2,9 +2,9 @@ HOSTS:
   master:
     roles:
       - master
-    platform: fedora-20-x86_64
+    platform: fedora-21-x86_64
```

Comment:
Since fedora 21 is EOL maybe bump to fedora 23? Also, I don't know where strings is tested in jenkins, but does this need a matching ci-job-configs change?